backend/controller/user.go

```diff
@@ -138,14 +148,25 @@ func CreateUser(w http.ResponseWriter, r *http.Request) {
 
 func DeleteUser(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	err, uId := helpers.ParseInt(w, r, "userId")
+	userIds := struct {
+		Users []uint `json:"users"`
+	}{}
+
+	err := json.NewDecoder(r.Body).Decode(&userIds)
 	if err != nil {
+		http.Error(w, "Malformed request", http.StatusBadRequest)
+	}
+
+	res := db.DB.Unscoped().Select(clause.Associations).Delete(&models.User{}, userIds.Users)
+	if res.Error != nil {
+		dbHelpers.DBErrorHandling(res.Error, w)
 		return
 	}
-	err = dbHelpers.DeleteObjandAssociationsByPk(w, models.User{Id: uId})
-	if err != nil {
+	if res.RowsAffected < 1 {
```

Comment:
should be erroring if we have less than inputted ids, no?

Reply:
resolved